src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureDraw, FigureArea {
+    private Colors color;

```

Comment:
let's create a constructor that accepts color and use `super` keyword to use this constructor in child figures